tests/unit/utils/test_file_utils.py

```diff
@@ -159,10 +159,10 @@ def test_with_permissions(self, mock_chown):
         file_utils.link_or_copy("foo/2", "qux/2", permissions=permissions)
 
         # Check that the copied file has the correct permission bits and ownership
-        assert stat.S_IMODE(os.stat("qux/2").st_mode) == 0o755
+        assert stat.S_IMODE(os.stat("qux/2").st_mode) == 0o755  # noqa: PLR2004
```

Comment:
I'm not asking for action here, but I opened a discussion about these: https://github.com/canonical/starbase/issues/148